### PR TITLE
Make environment delete button accessible

### PR DESCRIPTION
**Pull request by rebeccaalpert**
Made the environment delete button into a real button so it's in the tab flow and can be triggered by hitting return on the keyboard. Made the drag and drop button into a button as well so the styling would be consistent, but it's not in the tab flow since you can't drag and drop with the keyboard. (Let me know if you want it in the tab flow.)

Both now have a blue hover/focus state and blue focus ring since they're buttons.

Tabbing in action:
env-accessibility

I had to make some changes to the padding, etc. since the switch from spans to buttons threw them off.

Fixes [CONSOLE-1079](https://jira.coreos.com/projects/CONSOLE/issues/CONSOLE-1079).

@openshift/team-ux-review, could you please review? Let me know if you want the drag and drop button in the tab flow even though you can't use it with the keyboard.

### CODE REVIEW

**Comment by beanh66** [comment]
@serenamarie125 If we make the reordering capability accessible, that would be my expectation as well. Not sure how difficult that is from a dev perspective though @spadgett/ @rebeccaalpert?

Everything else looks great 👍

File: frontend/public/components/utils/name-value-editor.jsx [hunk 1]
```diff
@@ -67,8 +67,8 @@ export const NameValueEditor = withDragDropContext(class NameValueEditor extends
     return <React.Fragment>
       <div className="row">
         {!readOnly && allowSorting && <div className="col-xs-1 co-empty__header"></div>}
-        <div className="col-xs-5 text-secondary text-uppercase">{nameString}</div>
-        <div className="col-xs-5 text-secondary text-uppercase">{valueString}</div>
+        <div className="col-xs-5 text-secondary text-uppercase co-full__header">{nameString}</div>
```
Comment (spadgett) [inline frontend/public/components/utils/name-value-editor.jsx]: I don't see where `co-full__header` is defined?

**Comment by spadgett** [comment]
> what are your thoughts on whether the drag & drop control should be in the tab order? If it was, and hitting enter put you into reorder mode, i think that the arrow keys could move items and up down. Unsure if we need to go to this extent.

It would be a big effort since it's not supported in react-dnd, the library we're using. It's also an a really infrequent action and possible through editing the YAML if needed (or removing and adding a value again).

File: frontend/public/components/utils/_name-value-editor.scss [hunk 2]
```diff
@@ -1,5 +1,11 @@
 .co-empty__header {
-  width: auto;
+  padding: 0;
+  width: 28px;
```
Comment (rhamilto) [inline frontend/public/components/utils/_name-value-editor.scss]: Would love to see the value as a var so it does not have to be repeated below.

File: frontend/public/components/utils/_name-value-editor.scss [hunk 3]
```diff
@@ -43,11 +49,11 @@
 
 .pairs-list__side-btn {
   cursor: pointer;
-  padding-right: 8px;
 }
 
 .pairs-list__action-icon {
-  padding-right: 0;
+  color: $color-pf-black-900;
+  margin-left: 8px;
```
Comment (rhamilto) [inline frontend/public/components/utils/_name-value-editor.scss]: Would love to see the value as a var so it does not have to be repeated below.

**Comment by serenamarie125** [comment]
> It would be a big effort since it's not supported in react-dnd, the library we're using. It's also an a really infrequent action and possible through editing the YAML if needed (or removing and adding a value again).

ok then keeping it out of the tab flow makes sense.  Thanks @rebeccaalpert !

File: frontend/public/components/utils/_name-value-editor.scss (hunk 3 above)
Comment (rebeccaalpert) [reply frontend/public/components/utils/_name-value-editor.scss]: The other one is negative. I can make them both variables if you want, but they wouldn't be reused.

File: frontend/public/components/utils/name-value-editor.jsx (hunk 1 above)
Comment (rebeccaalpert) [reply frontend/public/components/utils/name-value-editor.jsx]: I think that's leftover cruft from an earlier go at restyling. Removed it.

**Comment by rebeccaalpert** [comment]
Addressed PR feedback.

File: frontend/public/components/utils/_name-value-editor.scss (hunk 3 above)
Comment (rhamilto) [reply frontend/public/components/utils/_name-value-editor.scss]: You can make a positive var negative.  e.g., `-($var)`

**Comment by rebeccaalpert** [comment]
jenkins rebuild

**Pull request merged**